tests/validator/validator.cpp

```diff
@@ -1995,14 +2004,13 @@ TEST(Validator, unitIllDefinedEquivalentUnits)
 
     validator->validateModel(m);
 
-    EXPECT_EQ_ERRORS(expectedErrors, validator);
+    EXPECT_EQ_ISSUES(expectedIssues, validator);
 }
 
 TEST(Validator, unitStandardUnitsWhichAreBaseUnits)
 {
-    const std::vector<std::string> expectedErrors = {
-        "Variable 'v1' has units of 'metre' and an equivalent variable 'v2' with non-matching units of 'second'. The mismatch is: metre^1, second^-1.",
-    };
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'v1' in component 'c1' has units of 'metre' and an equivalent variable 'v2' in component 'c2' with non-matching units of 'second'. The mismatch is: metre^1, second^-1."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done.